packages/Badge/src/index.tsx

```diff
@@ -1,32 +1,24 @@
-import React, { FunctionComponent } from "react";
+import React, { Fragment, FunctionComponent } from "react";
+
 interface IBadgeProps {
   type?: string;
-  to?: string;
-  round?: boolean;
   link?: boolean;
-  pill?: boolean;
   icon?: boolean;
-  children?: string | JSX.Element;
+  color?: string;
+  children?: any;
 }
 const Badge: FunctionComponent<IBadgeProps> = ({
   children,
   type,
-  pill,
+  color,
   icon,
-  round,
-  to,
   link,
   ...attrs
 }) => {
-  const assignType = type ? `badge--${type}` : "";
-  const isPill = pill ? "badge--pill" : "";
-  const isRound = round ? "badge--round" : "";
   const withIcon = icon ? "badge--icon-text" : "";
-  const classes = `badge ${assignType} ${isRound} ${isPill} ${withIcon}`;
+  const classes = `badge ${type ? `badge--${type}` : ""} ${withIcon} ${color}`;
```

Comment:
As suggestion, move type to the end or if type is empty will end up with an empty space at className. Not a big deal, just a small detail.

Reply:
Fixed